Library/RemoteConfig/RemoteConfigFeature+HelpersTests.swift

```diff
@@ -3,111 +3,102 @@ import Prelude
 import XCTest
 
 final class RemoteConfigFeatureHelpersTests: TestCase {
-  func testBlockUsers_RemoteConfig_FeatureFlag_False() {
+  func assertFeatureIsFalse(_ featureFlag: Bool,
+                            whenRemoteConfigFeatureIsFalse feature: RemoteConfigFeature) {
     let mockRemoteConfigClient = MockRemoteConfigClient()
-      |> \.features .~ [RemoteConfigFeature.blockUsersEnabled.rawValue: false]
+      |> \.features .~ [feature.rawValue: false]
 
     withEnvironment(remoteConfigClient: mockRemoteConfigClient) {
-      XCTAssertFalse(featureBlockUsersEnabled())
+      XCTAssertFalse(featureFlag)
     }
   }
 
-  func testBlockUsers_RemoteConfig_FeatureFlag_True() {
+  func assertFeatureIsTrue(_ featureFlag: Bool, whenRemoteConfigFeatureIsTrue feature: RemoteConfigFeature) {
     let mockRemoteConfigClient = MockRemoteConfigClient()
-      |> \.features .~ [RemoteConfigFeature.blockUsersEnabled.rawValue: true]
+      |> \.features .~ [feature.rawValue: true]
 
     withEnvironment(remoteConfigClient: mockRemoteConfigClient) {
-      XCTAssertTrue(featureBlockUsersEnabled())
+      XCTAssertFalse(featureFlag)
```

Comment:
I don't think these tests are working - we should pass in the `featureFlag` as a function instead so that it can be evaluated after the remoteConfig has been updated. In particular, I think this test should `XCTAssertTrue(featureFlag)` instead, and the fact that it passes the way it's written is concerning.

Also, if we want less boilerplate, thoughts on combining these into one function? Ie a `testFeatureMatchesRemoteConfig` and have it test both the true and the false case? I'm okay with them staying separate, though!

Reply:
Hahaha, OK, excellent catch here - on both bugs 😆  I see what you mean. Let me take another stab at this.

Reply:
As for combining the tests into one method - I just tried it out but honestly, all the closures make it look kind of ugly 😅 So just leaving them in separate test methods for now.